flang/lib/Optimizer/Transforms/RaiseToAffine.cpp

```diff
@@ -9,40 +9,116 @@
 #include "PassDetail.h"
```

Comment:
I'm assuming, that we'll probably want this file to be renamed to `AffinePromotion.cpp` to match the header.